<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BKMWidgetClass</class>
 <widget class="QWidget" name="BKMWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>428</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Constantia</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>BackupManagerWindow</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="autoSaveBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>启用自动保存</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string extracomment="start autobackup"/>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="backupItemList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::CopyAction</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="quickSaveBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>103</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Quick Save</string>
   </property>
   <property name="text">
    <string extracomment="quick backup">QS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quickLoadBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>103</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Quick Load</string>
   </property>
   <property name="text">
    <string extracomment="quick rollback">QL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="myPicture">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>121</width>
     <height>121</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Load</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="loggerEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="branchModeBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>启用自动保存</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string extracomment="start autobackup"/>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="backupFileFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BackupManagerQt64.qrc"/>
 </resources>
 <connections/>
</ui>
